.circleci/config.yml

```diff
@@ -306,6 +353,8 @@ workflows:
     jobs:
       - build-base
       - lint-build-debug
+      - sanitizer-build-debug
+      - clang-build-debug
```

Comment:
Do we need to enable _both_ workflows here? We're also trying to reduce our CI build times (see https://github.com/mapbox/valhalla-internal/issues/763) so wanted to understand if both of these are mandatory.

Reply:
No, we certainly do not. Sorry, this PR became kind of mess while I was trying to figure out how to get CI working